### ISSUE TITLE
About the learning rate of RoBerta in the finetuning stage?

### ISSUE THREAD

**Issue by zhengwsh** [ymcui/Chinese-BERT-wwm#75]
Can you provide the best learning rate for different tasks with Roberta? I can not find this in the technical report.

**Comment by ymcui** [comment]
Please check the table here:
https://github.com/ymcui/Chinese-BERT-wwm/blob/master/README_EN.md#baselines
